src/SFML/Audio/SoundFileWriterWav.cpp

```diff
@@ -137,8 +137,7 @@ bool SoundFileWriterWav::writeHeader(unsigned int sampleRate, unsigned int chann
     m_file.write(mainChunkId, sizeof(mainChunkId));
 
     // Write the main chunk header
-    Uint32 mainChunkSize = 0; // placeholder, will be written later
-    encode(m_file, mainChunkSize);
+    encode(m_file, static_cast<Uint32>(0));
```

Comment:
I would keep the comment here about 0 being a temporary placeholder.

Reply:
Fixed